### PR TITLE
Create /ebooks-and-whitepapers

### PR DESCRIPTION
**Pull request by Yndira-E**
## Description

This PR adds a new page that contains a link to all of our ebooks and whitepapers.

## Related Issue(s)

closes [#2773](https://github.com/FlowFuse/website/issues/2773)

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: .eleventy.js [hunk 1]
```diff
@@ -619,6 +619,19 @@ module.exports = function(eleventyConfig) {
         return nav;
     });
 
+    eleventyConfig.addCollection("ebooksAndWhitepapers", function(collectionApi) {
```
Comment (ZJvandeWeg) [inline .eleventy.js:622]: Let's call it a library or something else, but this is not so nice

File: src/ebooks-and-whitepapers/index.njk [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
```
Comment (ZJvandeWeg) [inline src/ebooks-and-whitepapers/index.njk:1]: REST wise this is not a great URL

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/ebooks-and-whitepapers/index.njk (hunk 2 above)
Comment (Yndira-E) [reply src/ebooks-and-whitepapers/index.njk:1]: how about `/resources/publications/`?

File: .eleventy.js (hunk 1 above)
Comment (Yndira-E) [reply .eleventy.js]: I've changed it to `publications`

**Review by Hasmin-AC — APPROVED** [review]

File: .eleventy.js [hunk 3]
```diff
@@ -619,6 +619,19 @@ module.exports = function(eleventyConfig) {
         return nav;
     });
 
+    eleventyConfig.addCollection("publications", function(collectionApi) {
+        return [
+            ...collectionApi.getFilteredByTag("whitepaper").map(item => {
```
Comment (ZJvandeWeg) [inline .eleventy.js:624]: Adding two collections and doing a single map on te elements would have been nicer IMO

**Review by ZJvandeWeg — APPROVED** [review]

**Comment by ZJvandeWeg** [comment]
@Yndira-E can you fix the conflicts?

**Pull request merged**